pandas/errors/__init__.py

```diff
@@ -320,9 +320,9 @@ class SettingWithCopyWarning(Warning):
     """
 
 
-class ChainedAssignmentError(ValueError):
+class ChainedAssignmentError(Warning):
```

Comment:
Shouldn't this be renamed to `ChainedAssignmentWarning` since it's no longer an exception?

Reply:
I was first doing that, and then didn't because thinking "it's still warning for an error you made (erroneous code that will never work". 
But either way is fine for me. Maybe calling it "..Error" might have more effect on users thinking they need to fix it, rather than just ignoring warnings as many people often do.

Reply:
I think I also slightly prefer the error here, this is an error on the user side.

I am also hoping that we can make this a real error sometime in the future